src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -1215,23 +1215,26 @@ abstract class RefChecks extends Transform {
       finally popLevel()
     }
 
+    private def showCurrentRef: String = {
+      val refsym = currentLevel.refsym
+      s"$refsym defined on line ${refsym.pos.line}"
+    }
+
     def transformStat(tree: Tree, index: Int): Tree = tree match {
       case t if treeInfo.isSelfConstrCall(t) =>
         assert(index == 0, index)
         try transform(tree)
         finally if (currentLevel.maxindex > 0) {
-          // An implementation restriction to avoid VerifyErrors and lazyvals mishaps; see scala/bug#4717
-          debuglog("refsym = " + currentLevel.refsym)
```

Comment:
Funny that it was considered useful to `debuglog` but not reveal to the user

Reply:
Funny ha-ha or funny just sad.

Reply:
Open to interpretation